src/components/crowdsale/utils.js

```diff
@@ -358,13 +358,16 @@ const getRate = async (addr, execID, methods) => {
 
 const calculateMinContribution = async (method, decimals, naturalMinCap, isWhitelisted) => {
   //todo: update for Proxy
-  const { minimum_contribution, minimum_purchase_amt, max_tokens_remaining, max_purchase_remaining } = await method.call()
-  const minimumContribution = toBigNumber(minimum_contribution).times(`1e-${decimals}`)
+  const { tier_min, minimum_contribution, minimum_purchase_amt, max_tokens_remaining, max_purchase_remaining } = await method.call()
+  const minimumContributionDutch = toBigNumber(minimum_contribution).times(`1e-${decimals}`)
+  const minimumContributionMintedCapped = toBigNumber(tier_min).times(`1e-${decimals}`)
+  const minimumContribution = minimumContributionMintedCapped || minimumContributionDutch
```

Comment:
Still is returning `naturalMinCap` as `toBigNumber` is returning a bigNumber object with value `0` when the parameter passed is not a number or a string representation of a number.

A solution would be to replace lines `362..364` with:
```
const minimumContribution = toBigNumber(tier_min || minimum_contribution).times(`1e-${decimals}`)
```

But I think that after this, in a different PR, we should replace `toBigNumber` signature and return `undefined` when an invalid value is passed.

What you think @vbaranov ?

Reply:
Yes, agree. It would be nice to mark invalid values as `undefined`